src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -27,7 +27,7 @@
  */
 public class ParserUtil {
 
-    public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
+    public static final String MESSAGE_INVALID_INDEX = "Index should be a non-zero positive integer.";
```

Comment:
Do we allow them to supply index as +1, +5, etc?
If we do not allow the plus sign I think unsigned sounds clearer

Reply:
Ok will change this